images/cilium/Dockerfile

```
@@ -113,3 +127,12 @@ ENV DEBUG_HOLD=${DEBUG_HOLD}
 COPY --from=builder /tmp/install/${TARGETOS}/${TARGETARCH}/usr/bin/cilium-agent /usr/bin/cilium-agent-bin
 COPY --from=builder /go/bin/dlv /usr/bin/dlv
 COPY --from=builder /out/${TARGETOS}/${TARGETARCH}/bin/debug-wrapper /usr/bin/cilium-agent
+
+# Copy in the debug symbols in case the binaries were stripped
+COPY --from=builder /tmp/debug/${TARGETOS}/${TARGETARCH} /usr/lib/debug
```

This is where `dlv` and `gdb` will look for the symbols: https://sourceware.org/gdb/current/onlinedocs/gdb.html/Separate-Debug-Files.html 

```suggestion
COPY --from=builder /tmp/debug/${TARGETOS}/${TARGETARCH} /usr/lib/debug/usr/bin
```

---

We need to preserve the hierarchy in a generic way because some binaries land outside `/usr/bin`. Working on a solution.

---

Done.